lib/private/Contacts/ContactsMenu/Manager.php

```diff
@@ -59,14 +59,14 @@ public function __construct(ContactsStore $store, ActionProviderStore $actionPro
 
 	/**
 	 * @param IUser $user
-	 * @param string $filter
+	 * @param string|null $filter
 	 * @return array
 	 */
 	public function getEntries(IUser $user, $filter) {
 		$maxAutocompleteResults = max(0, $this->config->getSystemValueInt('sharing.maxAutocompleteResults', Constants::SHARING_MAX_AUTOCOMPLETE_RESULTS_DEFAULT));
 		$minSearchStringLength = $this->config->getSystemValueInt('sharing.minSearchStringLength', 0);
 		$topEntries = [];
-		if (strlen($filter) >= $minSearchStringLength) {
+		if ($filter && strlen($filter) >= $minSearchStringLength) {
```

Comment:
```suggestion
		if (strlen($filter ?? '') >= $minSearchStringLength) {
```
I would avoid using $filter as a bool because of usual side effects ('0' evaluates to false)

Reply:
And on top of that that prevented to get entries. :see_no_evil: